sfge/src/main/java/com/salesforce/config/UserFacingMessages.java

```diff
@@ -94,21 +94,30 @@ public static final class MultipleMassSchemaLookupRuleTemplates {
          * call as well at a different line.
          */
         public static final String PRECEDED_BY_MESSAGE_TEMPLATE =
-                "was preceded by another expensive schema lookup";
+                "Multiple expensive schema lookups are invoked";
         /**
          * Indicates when the same expensive schema call is invoke multiple times through various
          * method calls.
          */
         public static final String CALL_STACK_TEMPLATE =
-                "was invoked more than once in the call stack";
+                "executed multiple times in the call stack";
 
         /**
          * String Param 1: Expensive Schema Lookup type (Schema.getGlobalDescribe or
-         * Schema.describeSObjects) String Param 2: Text corresponding to occurrence type ({@link
-         * #LOOP_MESSAGE_TEMPLATE}, {@link #PRECEDED_BY_MESSAGE_TEMPLATE}, {@link
-         * #CALL_STACK_TEMPLATE}) String Param 3: Occurrence information using {@link
-         * #OCCURRENCE_TEMPLATE}. Can be more than one occurrence.
+         * Schema.describeSObjects)
+         *
+         * <p>String Param 2: Text corresponding to occurrence type ({@link #LOOP_MESSAGE_TEMPLATE},
+         * {@link #PRECEDED_BY_MESSAGE_TEMPLATE}, {@link #CALL_STACK_TEMPLATE})
+         *
+         * <p>String Param 3: Occurrence information using {@link #OCCURRENCE_TEMPLATE}. Can be more
+         * than one occurrence.
          */
         public static final String MESSAGE_TEMPLATE = "%1$s %2$s. %3$s";
+
+        /**
+         * Exact same arguments as {@link #MESSAGE_TEMPLATE}, except the first argument is unused.
+         * Meant for violation messages that don't include the name of the expensive method.
+         */
+        public static final String METHODLESS_MESSAGE_TEMPLATE = "%2$s. %3$s";
```

Comment:
The approved variant of one of the violation messages doesn't have the method name at the beginning, so I just created an alternate template variant that doesn't use the first argument. That allowed me to maximize code reuse.